src/PlusDataCollection/Andor/vtkPlusAndorCamera.h

```diff
@@ -0,0 +1,266 @@
+  int GetReadMode();
+
+  /*! Trigger mode. Valid values:
+   * 0. Internal
+   * 1. External
+   * 6. External Start
+   * 7. External Exposure (Bulb)
+   * 9. External FVB EM (only valid for EM Newton models in FVB mode)
+   * 10. Software Trigger
+   * 12. External Charge Shifting
+   */
+  PlusStatus SetTriggerMode(int triggerMode);
+  int GetTriggerMode();
+
+  /*! Normal operating temperature (degrees celsius). */
+  PlusStatus SetCoolTemperature(int coolTemp);
+  int GetCoolTemperature();
+
+  /*! Lowest temperature at which it is safe to shut down the camera. */
+  PlusStatus SetSafeTemperature(int safeTemp);
+  int GetSafeTemperature();
+
+  /*! Get the current temperature of the camera in degrees celsius. */
+  float GetCurrentTemperature();
+
+  /*! Paths to additive and multiplicative bias correction images. */
+  PlusStatus SetBiasCorrectionImage(std::string biasFilePath);
+  std::string GetBiasCorrectionImage()
+  {
+    return biasCorrection;
+  }
+  PlusStatus SetFlatCorrectionImage(std::string flatFilePath);
+  std::string GetFlatCorrectionImage()
+  {
+    return flatCorrection;
+  }
+
+  /*! -1 uses currently active settings. */
+  PlusStatus AcquireBLIFrame(int binning, int vsSpeed, int hsSpeed, float exposureTime);
+
+  /*! -1 uses currently active settings. */
+  PlusStatus AcquireGrayscaleFrame(int binning, int vsSpeed, int hsSpeed, float exposureTime);
+
+  /*! Convenience function to save a bias frame for a certain binning/speed configuration. */
+  PlusStatus AcquireBiasFrame(std::string biasFilePath, int binning, int vsSpeed, int hsSpeed);
+
+  /*! Cooler Mode control. When CoolerMode is set on, the cooler
+      will be kept online when the camera is shutdown. This is helpful to
+      reduce the cooling cycles the camera undergoes. Turning the Cooler ON
+      and OFF should be done sparingly for the same reason.
+  */
+  PlusStatus SetCoolerMode(int mode);
+
+  /*! Wait for the camera to reach operating temperature (e.g. -70°C). */
+  void WaitForCooldown();
+  /*! Wait for the camera to reach safe temperature for poweroff (e.g. 10°C). */
+  void WaitForWarmup();
+
+  /*! Check the return status of Andor SDK functions. */
+  unsigned int checkStatus(unsigned int returnStatus, std::string functionName);
+
+  vtkPlusAndorCamera(const vtkPlusAndorCamera&) = delete;
+  void operator=(const vtkPlusAndorCamera&) = delete;
+
+protected:
+  /*! Constructor */
+  vtkPlusAndorCamera();
+
+  /*! Destructor */
+  ~vtkPlusAndorCamera();
+
+  /*! Device-specific connect */
+  virtual PlusStatus InternalConnect();
+
+  /*! Device-specific disconnect */
+  virtual PlusStatus InternalDisconnect();
+
+  /*! Device-specific recording start */
+  PlusStatus InternalStartRecording() override;
+
+  /*! Device-specific recording stop */
+  PlusStatus InternalStopRecording() override;
+
+  /*! Initialize vtkPlusAndorCamera */
+  PlusStatus InitializeAndorCamera();
+
+  using DataSourceArray = std::vector<vtkPlusDataSource*>;

```

Comment:
TIL a new using syntax. @lassoan @Sunderlandkyl Should this be moved to the top of the class declaration (similar to typedef's)?